### PR TITLE
Add get_transient_memory_usage() method to AttibuteInitializer.

### PR DESCRIPTION
**Pull request by toregge**
@geirst : please review

### CODE REVIEW

File: searchcore/src/tests/proton/attribute/attribute_initializer/attribute_initializer_test.cpp [hunk 1]
```diff
@@ -198,6 +212,22 @@ TEST("require that too old attribute is not loaded")
...
+}
+
+TEST("require that transient memory usage for normal posting list attribute load")
+{
+    saveAttr("a", get_int32_wset_fs(), 10, 2);
+    Fixture f;
+    auto avi = f.createInitializer({"a", get_int32_wset_fs()}, 5);
+    EXPECT_EQUAL(24u, avi->get_transient_memory_usage());
+}
+
```
Comment (geirst) [inline searchcore/src/tests/proton/attribute/attribute_initializer/attribute_initializer_test.cpp:230]: Could also consider testing that transient memory is zero for a non fast-search attribute.

File: searchlib/src/vespa/searchlib/attribute/attribute_header.h [hunk 2]
```diff
@@ -63,6 +63,8 @@ class AttributeHeader {
     bool getEnumerated() const { return _enumerated; }
     uint64_t getCreateSerialNum() const { return _createSerialNum; }
     uint32_t getVersion() const  { return _version; }
+    uint64_t get_total_value_count() const { return _totalValueCount; }
```
Comment (geirst) [inline searchlib/src/vespa/searchlib/attribute/attribute_header.h:66]: Please test these two in verify_roundtrip_serialization() in attribute_header_test.cpp.

**Review by geirst — APPROVED** [review]
👍 Please consider my comments.

**Review by geirst — APPROVED** [review]
👍
